openstack/nova/templates/bin/_queens-dry-run.tpl

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+set -e
+set -x
+uname
+whoami
+
+mkdir -p /var/lib/apt/lists/partial
+apt update || true
+apt-get -y install postgresql-client postgresql || true
+export PGPASSWORD="{{.Values.postgresql.postgresPassword}}"
+PATH=$PATH:/usr/lib/postgresql/9.5/bin
+export PATH
+
+echo " - Dumping running database"
+mkdir -p /tmp/novadb
+pg_dump -U {{.Values.postgresql.user}} -h {{.Chart.Name}}-postgresql.{{include "svc_fqdn" .}} nova > /tmp/nova.sql
+pg_dump -U {{.Values.postgresql.user}} -h {{.Chart.Name}}-postgresql.{{include "svc_fqdn" .}} nova_api > /tmp/nova_api.sql
+
+sudo -u postgres mkdir -p /var/run/postgresql/9.5-main.pg_stat_tmp/
+sudo -u postgres /usr/lib/postgresql/9.5/bin/pg_ctl -D /etc/postgresql/9.5/main start
+sleep 5
+sudo -u postgres createuser -s root
+psql postgres -c "CREATE USER nova WITH PASSWORD 'test';"
+psql postgres -c "CREATE USER nova_API WITH PASSWORD 'test';"
+createdb -O nova nova
+createdb -O nova_api nova_api
+psql nova < /tmp/nova.sql
+psql nova_api < /tmp/nova_api.sql
+
+
+mkdir -p /etc/nova
+cat <<EOT >> /etc/nova/nova.conf
+[DEFAULT]
+
+[database]
+connection = postgresql+psycopg2://nova:test@localhost/nova
+[api_database]
+connection = postgresql+psycopg2://nova_api:test@localhost/nova_api
```

Comment:
Does postgres not care for upper/lower-case in usernames? Because the user is created as `nova_API`.

Reply:
oh you're right, typo. but did seemed to work